hackathon_site/event/serializers.py

```diff
@@ -96,27 +97,36 @@ def create(self, validated_data):
         if hasattr(current_user, "profile"):
             raise serializers.ValidationError("User already has profile")
 
-        try:
-            rsvp_status = Application.objects.get(user=current_user).rsvp
-            if not rsvp_status:
+        is_test_user = (
+            self.context["request"]
+            .user.groups.filter(name=settings.TEST_USER_GROUP)
+            .exists()
+        )
+
+        if not is_test_user:
+            try:
+                rsvp_status = Application.objects.get(user=current_user).rsvp
+                if not rsvp_status and settings.RSVP:
+                    raise serializers.ValidationError(
+                        "User has not RSVP'd to the hackathon. Please RSVP to access the Hardware Signout Site"
+                    )
+            except Application.DoesNotExist:
                 raise serializers.ValidationError(
-                    "User has not RSVP'd to the hackathon. Please RSVP to access the Hardware Signout Site"
+                    "User has not completed their application to the hackathon. Please do so to access the Hardware Signout Site"
                 )
-        except Application.DoesNotExist:
-            raise serializers.ValidationError(
-                "User has not completed their application to the hackathon. Please do so to access the Hardware Signout Site"
-            )
 
-        try:
-            review_status = Review.objects.get(application__user=current_user).status
-            if review_status != "Accepted":
+            try:
+                review_status = Review.objects.get(
+                    application__user=current_user
+                ).status
+                if review_status != "Accepted":
+                    raise serializers.ValidationError(
+                        "User has not been accepted to participate in hackathon"
```

Comment:
Change to "User has not been accepted to participate in **the** hackathon"

Reply:
done